ocaml/idl/datamodel_host.ml

```diff
@@ -373,6 +374,13 @@ let evacuate =
         ; param_release= next_release
         ; param_default= Some (VRef null_ref)
         }
+      ; {
+          param_type= Int
+        ; param_name= "evacuate_batch_size"
+        ; param_doc= "The maximum number of VMs to be migrated per batch"
+        ; param_release= next_release
+        ; param_default= None
```

Comment:
From https://github.com/xapi-project/xen-api/blob/master/ocaml/idl/ocaml_backend/gen_server.ml#L67, I understand is that `None` means it is a mandatory parameter.
While this should not be as the old clients may send calls without the new parameter. Then `Some <N>` would be a default value in this case.